standalone/gui/application.cpp

```diff
@@ -57,6 +57,43 @@ auto application::make_dial(int index)
     return align_center_middle(hmargin({20.,20.},markers));
 }
 
+auto application::make_toggle_button(size_t index, std::string const &label_name, auto color) {
```

Comment:
Dentro de arquivos .cpp, que são compilados, é melhor substituir o `auto` pelo tipo do `color` mesmo. 

Isso garante que a interface vai ser gerada mesmo que não for usada e facilita as recomendações da IDE. Sem isso, você pode ter um erro de símbolo ausente no futuro.

O `auto` e templates são mais reservados para arquivos .h.